src/simpleDetFunc.c

```diff
@@ -263,9 +264,9 @@ static inline float calcPSMEntryContrib(float* x, int window_size, float sigma)
 	return out;
 }
 
-void pSMContribution(int correntropyWinSize, int interval,
-		     int numWindows, float *buffer,
-		     float *sigmas,float **pSMatrix)
+int pSMContribution(int correntropyWinSize, int interval,
```

Comment:
Ditto on return value

Reply:
Same as my response for centralFreqMapper - it has been fixed